starter-code/css/main.css

```diff
@@ -6,3 +6,189 @@ Green: #00B172
 White: #FFF
 
 */
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+*{
+  margin: 0;
+  padding: 0;
+  font-family: 'Open Sans';
+}
+
+nav {
+  position: fixed;
+  width: 100%;
+  height: 60px;
+  margin: 0 auto;
+}
+
+nav div {
+  float: left;
+}
+
+nav div img {
+  width: 200px;
+  height: auto;
+  margin: 25px 25px auto;
+}
+
+nav ul {
+  float: right;
+  margin: 50px 50px auto;
+}
+
+nav ul li{
+  list-style-type: none;
+  display: inline;
+  margin-left: 25px;
+  font-size: 20px;
+}
+
+.container {
+  padding-top: 51px;
+}
+
+header {
+  margin: 60px 0px 20px;
+  background-image: url("../images/landing.jpg");
+  background-position: 0 0;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 750px;
+  position: relative;
+}
+
+h1 {
+  font-size: 4em;
+  color: white;
+  padding: 250px 500px 80px;
```

Comment:
You are centering the title manually. It may look great on your computer but it's not guaranteed to be the case for others.